pkg/drivers/kic/oci/oci.go

```diff
@@ -51,9 +53,18 @@ func DeleteAllContainersByLabel(ociBin string, label string) []error {
 		return nil
 	}
 	for _, c := range cs {
-		cmd := exec.Command(ociBin, "rm", "-f", "-v", c)
-		if out, err := cmd.CombinedOutput(); err != nil {
-			deleteErrs = append(deleteErrs, errors.Wrapf(err, "delete container %s: output %s", c, out))
+		_, err := containerStatus(ociBin, c)
```

Comment:
Is there a command available to detect this that does not require querying the status of an individual container?

It seems like it would be better to detect a stuck daemon in a general fashion (like `docker status` or some other command), rather than one that could be impacted by a container not existing.

Reply:
unfortunately that is not the case ! docker status works great ! and it even lets your create new containers

and everything is normal ! only this container is stuck and u can not do anything to it

that was only way I could detect it.